client/components/domains/search-filters/tld-filter.jsx

```diff
@@ -0,0 +1,113 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { includes } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FormFieldset from 'components/forms/form-fieldset';
+import Popover from 'components/popover';
+import TokenField from 'components/token-field';
+
+export class TldFilterControl extends Component {
+	static propTypes = {
+		availableTlds: PropTypes.arrayOf( PropTypes.string ).isRequired,
+		onChange: PropTypes.func.isRequired,
+		onFiltersReset: PropTypes.func.isRequired,
+		onFiltersSubmit: PropTypes.func.isRequired,
+		tlds: PropTypes.arrayOf( PropTypes.string ).isRequired,
+	};
+
+	state = {
+		showPopover: false,
+	};
+
+	togglePopover = () => {
+		this.setState( {
+			showPopover: ! this.state.showPopover,
+		} );
+	};
+
+	handleFiltersReset = () => {
+		this.togglePopover();
+		this.props.onFiltersReset( 'tlds' );
+	};
+	handleFiltersSubmit = () => {
+		this.togglePopover();
+		this.props.onFiltersSubmit();
+	};
+	handleOnChange = newTlds => {
+		this.props.onChange(
+			'tlds',
+			newTlds.filter( tld => includes( this.props.availableTlds, tld ) )
+		);
+	};
+
+	bindButton = button => ( this.button = button );
+
+	render() {
+		const { tlds, translate } = this.props;
+		const hasFilterValue = tlds.length > 0;
+		return (
+			<div className="search-filters__filter search-filters__tld-filter">
+				<Button
+					className={ classNames( { 'is-active': hasFilterValue } ) }
+					onClick={ this.togglePopover }
+					ref={ this.bindButton }
+				>
+					{ translate( 'Extensions', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Woo Extensions' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
That's pretty bad, bot.